# Define the goal
uint32 grasp_id
---
# Define the result
uint32 total_grasped
---
# Define a feedback message
float32 percent_complete
